public/video-ui/src/components/VideoSelectBar/VideoSelectBar.js

```diff
@@ -10,7 +10,7 @@ export default class VideoSelectBar extends React.Component {
 
     return (
       <div className="bar info-bar">
-        <span className="bar__message">You are in an embedded version of the media atom maker</span>
+        <span className="bar__message"></span>
```

Comment:
I think we should remove the empty span for now, it can always be added again if needed later.